model/src/main/kotlin/AdvisorResult.kt

```diff
@@ -23,17 +23,17 @@ package org.ossreviewtoolkit.model
  * The result of a single vulnerability check of a single package.
  */
 data class AdvisorResult(
-    /**
+        /**
      * The found vulnerabilities.
      */
-    val vulnerability: List<Vulnerability>,
+    val vulnerabilities: List<Vulnerability>,
```

Comment:
How does changing this to the plural form fix the issue? (As this is the commit that closes #3376.)

Reply:
I just did this since it was the last commit of the PR. So would you rather have the close phrase in the commit that really concludes the issue? Should I also have the plural form in a separate PR?

Reply:
> So would you rather have the close phrase in the commit that really concludes the issue?

Yes please. If a fix is done in a single commit, then the keyword should go to that commit. If a fix is spread across several (consecutive) commits, then the keyword should go to the last commit of that series, which not necessarily is the last commit in a PR.

> Should I also have the plural form in a separate PR?

No. That's still roughly the same topic, so it's completely fine to have it here. It's just that the commit is unrelated to the actual fix, so it should not have the closing keyword.